packages/ui/__stories__/Icon.stories.tsx

```diff
@@ -10,12 +11,24 @@ export default {
   component: Icon,
 }
 
-export const Normal = () => <Icon icon={Clipboard} ariaLabel="Icon copy to clipboard" />
+export const FeatherIconNormal = () => <Icon icon={Clipboard} ariaLabel="Icon copy to clipboard" />
```

Comment:
Could you remove the screenshots for the old stories?

Reply:
Sorry, thought it does that automatically.

Reply:
No worries. I think we should contribute that cleanup functionality to Loki at some point